### ISSUE TITLE
get_global no implemented

### ISSUE THREAD

**Issue by olafura** [ElixiumNetwork/WaspVM#17]
```
WaspVM.load_file(ref, "../AtomVM/src/AtomVM.wasm")
** (RuntimeError) Not implemented: :get_global in init expression
    (wasp_vm) lib/decoding/global_section_parser.ex:58: WaspVM.Decoder.GlobalSectionParser.evaluate_init_expr/2
    (wasp_vm) lib/decoding/global_section_parser.ex:33: WaspVM.Decoder.GlobalSectionParser.parse_entries/2
    (wasp_vm) lib/decoding/global_section_parser.ex:16: WaspVM.Decoder.GlobalSectionParser.parse/1
    (wasp_vm) lib/decoding/decoder.ex:53: WaspVM.Decoder.do_decode/2
    (wasp_vm) lib/wasp_vm.ex:41: WaspVM.load_file/2
```

**Comment by olafura** [comment]
[AtomVM.zip](https://github.com/ElixiumNetwork/WaspVM/files/2707837/AtomVM.zip)

**Comment by alexdovzhanyn** [comment]
Implemented in commits [2958052](https://github.com/ElixiumNetwork/WaspVM/commit/29580526938304872fbd4b4b7d0e898966a89d93) and [eca0d9e](https://github.com/ElixiumNetwork/WaspVM/commit/eca0d9e8e3fd2ac55c9154552fb0a1403a6a2966). Will be included in the next release once we're ready, I'll leave this issue open until then.

**Comment by alexdovzhanyn** [comment]
Fixed and released in [v0.8.0](https://github.com/ElixiumNetwork/WaspVM/releases/tag/v0.8.0)
